app/src/main/java/com/onegini/mobile/exampleapp/view/activity/AuthenticationActivity.java

```diff
@@ -44,8 +45,8 @@
   @SuppressWarnings({ "unused", "WeakerAccess" })
   @Bind(R.id.authenticator_message)
   TextView authenticatorMessage;
-
-  @SuppressWarnings({"unused"})
+  @Nullable
```

Comment:
I don't like adding this view when it's not used either, but it is already added to views like push with fingerprint or push with pin, while it's not used there. So we should remove it from classes that don't need it or add it to all classes - as long as it's consistent solution.